programs/monaco_protocol/src/instructions/market/create_market.rs

```diff
@@ -152,6 +152,10 @@ pub fn initialize_outcome(ctx: Context<InitializeMarketOutcome>, title: String)
         ctx.accounts.market.market_status == MarketStatus::Initializing,
         CoreError::MarketOutcomeMarketInvalidStatus
     );
+    require!(
+        title.len() <= MarketOutcome::TITLE_MAX_LENGTH,
+        CoreError::MarketOutcomeTitleTooLong
+    );
```

Comment:
NP might be nice to add an internal method we can unit test rather than adding more anchor tests